client/Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[package]
+name = "ame_client"
```

Comment:
Does the naming here make sense? it seems kind of the inconsistent with the other crates.